roles/contiv_network/tasks/main.yml

```diff
@@ -108,3 +108,8 @@
 
 - include: aci_tasks.yml
   when: (run_as == "master") and (contiv_network_mode == "aci")
+
+- name: set fwd mode
+  when: (run_as == "master")
```

Comment:
sorry to nit-pic, suggest to move `when` condition after `shell:`, to look consistent with rest of tasks.